plugin/src/main/java/net/dzikoysk/funnyguilds/feature/hooks/holographicdisplays/FunnyHologramManager.java

```diff
@@ -0,0 +1,16 @@
+package net.dzikoysk.funnyguilds.feature.hooks.holographicdisplays;
+
+import net.dzikoysk.funnyguilds.guild.Guild;
+import panda.std.Option;
+
+public interface FunnyHologramManager {
+
+    Option<FunnyHologram> createHologram(Guild guild);
+
+    void deleteHologram(Guild guild);
+
+    void deleteHologram(FunnyHologram hologram);
```

Comment:
Warto w delete zamiast voida, to np. dać boolean czy faktycznie coś usunięto, albo starąś wartość czy coś